### ISSUE TITLE
_layouts are excluded from git, yet are checked in

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#773]
The .ignore plugin in IntelliJ warns that there are some files under version control that are covered by .gitignore patterns.

These include

- _layouts/homepage.html
- _layouts/default.html

Sure enough, .gitignore includes the following entries:
```
# ignore github pages file and site.
_site/**
_layouts/**
Gemfile
```

Either
 a) these files are valid - in which case .gitignore needs updating to not exclude them 
OR
b) These files are not valid - in which case they should be deleted from source control

@jmertic - do you happen to know? I am not family with the website setup

**Comment by cmgrote** [comment]
I think that at least `_layouts` may need to be removed from `.gitignore`, as the website uses this to show the different layouts you see on http://egeria.odpi.org (ie. main page is different from sub-pages due to different layouts).

That said, I suspect these are needed in a specific branch that GitHub itself uses for displaying the website -- which may not be `master`...  Not sure that we could `.gitignore` from just master and leave them in the other branch, though (?)

**Comment by planetf1** [comment]
I've removed from .gitignore as these files appear to be valid for GitHub pages. PR open

**Comment by cmgrote** [comment]
@planetf1 just a heads-up, you are aware that you can describe the PR with a comment like `fixes #1234` or `resolves #1234` (where `1234` is the issue number), and when the PR is merged it will automatically close the issue as well?

That saves you needing to manually close things + doing so before the PR is merged 😉
